index.html

```diff
@@ -773,7 +773,7 @@
 			<li>If the pointer is not in the <a>active buttons state</a>, then terminate these steps.</li>
 			<li>For the specified <code>pointerId</code>, set the <dfn>pending pointer capture target override</dfn> to the <code>Element</code> on which this method was invoked.</li>
 		</ol>
-		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements.</div>
+		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements. The region of the capturing element includes the element itself and all its children to mimic the hover state of the capturing element. Moving out of this region dispatches <code>pointerout</code> only to the capturing element and <code>pointerleave</code> to the capturing element and all its ancestors as if the pointer has left the whole document. Moving into this region dispatches <code>pointerover</code> only to the capturing element and <code>pointerenter</code> to the capturing element and all its ancestors as if the pointer entered the capturing element from outside of the document.</div>
```

Comment:
One problem with the current (as well as the original) wording is that it implies that we don't expect the transition events when the border is _not_ crossed: the triggering of transition events is tied to "... generated when crossing..." and "Moving out of this region...".

We need to cover the case when the pointer is captured while the pointer is away from the captured node. In this case, we expect pointerleave/pointerout even when the pointer is stationary (i.e., no boundary crossing). Edge does the right thing here (check [this demo](https://output.jsbin.com/rupulu)).

I think mentioning any of the "interpretations" Navid posted above would let us avoid enumerating all the triggering scenarios. I find the first interpretation pretty intuitive.